packages/manager/modules/email-domain/src/index.js

```diff
@@ -49,30 +37,18 @@ angular
       $stateProvider.state('app.email-delegate', {
         url: '/email_delegate',
         template: '<div ui-view></div>',
-        redirectTo: 'app.email-delegate.index',
         resolve: {
           breadcrumb: /* @ngInject */ ($translate) =>
             $translate.instant('emails_domain_delegate_title'),
         },
       });
 
-      $stateProvider.state('app.email-delegate.index.**', {
-        url: '',
-        lazyLoad: ($transition$) => {
-          const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
-
-          return import('./email.module').then((mod) =>
-            $ocLazyLoad.inject(mod.default || mod),
-          );
-        },
-      });
-
       $stateProvider.state('app.email-delegate.dashboard.**', {
         url: '/:productId',
         lazyLoad: ($transition$) => {
           const $ocLazyLoad = $transition$.injector().get('$ocLazyLoad');
 
-          return import('./delegate/delegate.module').then((mod) =>
+          return import('./email.module').then((mod) =>
```

Comment:
I'm not sure the two have the same structure and that we need all email domain module for delegated email (it might need to be another module in another step)